### PR TITLE
IGAPP-202: Large font ios webview

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: native/src/modules/common/renderHtml.js [hunk 1]
```diff
@@ -147,6 +147,10 @@ const renderHtml = (html: string, files: PageResourceCacheStateType, theme: Them
         list-style-position: inside;
         \`} */
     }
+    
+    body * {
+        font-size: ${theme.fonts.contentFontSize};
```
Comment (maxammann) [inline native/src/modules/common/renderHtml.js:152]: Doesn't this set the font-size of every header ( "h1, h2" etc), element?

File: native/src/modules/common/renderHtml.js [hunk 2]
```diff
@@ -137,8 +137,8 @@ const renderHtml = (html: string, files: PageResourceCacheStateType, theme: Them
         margin: 0;
         padding: 0;
 
+        font: -apple-system-body;
```
Comment (maxammann) [inline native/src/modules/common/renderHtml.js:140]: Did you test whether this has any side effects on android?

File: native/ios/Integreat.xcodeproj/xcshareddata/xcschemes/integreat-test-cms.xcscheme [hunk 3]
```diff
@@ -27,6 +27,15 @@
             <ActionContent
                title = "Setup native locales"
                scriptText = "exec &gt; ${PROJECT_DIR}/prebuild_native_locales.log 2&gt;&amp;1&#10;cd ${PROJECT_DIR} &amp;&amp; ../tools/native-locales setup IntegreatTestCms --locales &quot;../../locales/locales.json&quot; --destination &quot;Integreat&quot;&#10;">
+               <EnvironmentBuildable>
+                  <BuildableReference
+                     BuildableIdentifier = "primary"
+                     BlueprintIdentifier = "13B07F861A680F5B00A75B9A"
+                     BuildableName = "IntegreatTestCms.app"
```
Comment (maxammann) [inline native/ios/Integreat.xcodeproj/xcshareddata/xcschemes/integreat-test-cms.xcscheme:34]: Is this change on purpose?

**Review by maxammann — CHANGES_REQUESTED** [review]

File: native/src/modules/common/renderHtml.js (hunk 2 above)
Comment (klinzo) [reply native/src/modules/common/renderHtml.js:140]: Yep, no side effects afaics. Feel free to test for yourself :)

File: native/ios/Integreat.xcodeproj/xcshareddata/xcschemes/integreat-test-cms.xcscheme (hunk 3 above)
Comment (klinzo) [reply native/ios/Integreat.xcodeproj/xcshareddata/xcschemes/integreat-test-cms.xcscheme:34]: Yes, otherwise the locales preaction script is not run.

Comment (klinzo) [reply native/ios/Integreat.xcodeproj/xcshareddata/xcschemes/integreat-test-cms.xcscheme:34]: Was still missing for IntegreatTestCms here, all other configs are already using it.

**Review by NKlug — APPROVED** [review]

**Review by maxammann — APPROVED** [review]

File: native/src/modules/common/renderHtml.js (hunk 1 above)
Comment (klinzo) [reply native/src/modules/common/renderHtml.js:155]: Fixed
